fds-lambda-ml-integration/src/main/java/com/provectus/fds/ml/InvokeEndpointLambda.java

```diff
@@ -10,33 +10,44 @@
 import com.amazonaws.services.sagemakerruntime.model.InvokeEndpointResult;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.StringJoiner;
 
+@SuppressWarnings("unused")
 public class InvokeEndpointLambda implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    private static final Logger logger = LogManager.getLogger(InvokeEndpointLambda.class);
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
 
     @Override
     public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent input, Context context) {
         try {
-            ObjectMapper objectMapper = new ObjectMapper();
             objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
do it on instance creation

Reply:
fixed in all the lambdas (current module undoubtedly)